xml/System.IO/FileSystemWatcher.xml

```diff
@@ -830,13 +830,42 @@
   
  :::code language="cpp" source="~/samples/snippets/cpp/VS_Snippets_CLR_Classic/classic NotifyFilters Example/CPP/source.cpp" id="Snippet1":::
  :::code language="csharp" source="~/samples/snippets/csharp/System.IO/FileSystemWatcher/Overview/source.cs" id="Snippet1":::
- :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_Classic/classic NotifyFilters Example/VB/source.vb" id="Snippet1":::  
+ :::code language="vb" source="~/samples/snippets/visualbasic/VS_Snippets_CLR_Classic/classic NotifyFilters Example/VB/source.vb" id="Snippet1":::
+
+## Troubleshooting
```

Comment:
Better place this in Remarks section above example, it's not in API Docs convention to create third `Troubleshooting` section